webapp/src/store/util.ts

```diff
@@ -60,3 +61,10 @@ export function wrapActions (actions: ActionTree<AppState, AppState>): ActionTre
     };
   }, {});
 }
+
+export function googleSignOut() {
```

Comment:
Have the `async` keyword be part of this function declaration

Reply:
Done!